scripts/ci-e2e.sh

```diff
@@ -31,7 +31,7 @@ source "${REPO_ROOT}/hack/ensure-kustomize.sh"
 # Configure provider images generation;
 # please ensure the generated image name matches image names used in the E2E_CONF_FILE
 export REGISTRY=gcr.io/k8s-staging-cluster-api
-export TAG=ci
```

Comment:
Why change the tag for the CI?

Reply:
moved to https://github.com/kubernetes-sigs/cluster-api/pull/3651